README.md

```diff
@@ -1,15 +1,15 @@
-# README
+# Hangman Web Application Game
 
-This is a simple web application game of Hangman
+This is a simple web application game of Hangman.
 
 In order to run this game on your local system, keep these specs and steps in mind:
 
-* Ruby versions 2.7.0 or later to run Rails
+* Ruby versions 2.7.0 or later is required to support Rails.
 
-* Use ```gem install rails``` to install rails 7.0.0 or later
+* Use ```gem install rails``` to install rails 7.0.0 or later, if not previously installed.
```

Comment:
`bundle install` installs all the dependencies (including rails), and that manages the specific versions of gems.  Therefore, this step is not necessary I think.